specification/purview/data-plane/readme.md

```diff
@@ -100,6 +100,18 @@ tag: package-2018-12-01-preview
 title: PurviewScanningClient
 ```
 
+### Tag: package-2023-09
+
+These settings apply only when `--tag=package-2023-09` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-09-01'
+input-file:
+  - Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json
+modelerfour:
+  lenient-model-deduplication: true
```

Comment:
Why you have this here?

duplication in model should better be solved, not suppressed (if need to suppress, it should be in each SDK).

Reply:
Sure, I just removed modelerfour from the PR